MaterialLib/PorousMedium/Permeability/DupuitPermeability.h

```diff
@@ -0,0 +1,61 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2018, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+
+#pragma once
+
+#include <Eigen/Dense>
+
+#include "BaseLib/Error.h"
+#include "MaterialLib/PorousMedium/Permeability/Permeability.h"
+#include "ParameterLib/Parameter.h"
+
+namespace MaterialLib
+{
+namespace PorousMedium
+{
+/// The class implements a basic permeability model that employs a parameter
+/// (for instance a constant parameter or mesh cell dependend parameter) to fill
+/// the intrinsic permeability tensor.
+class DupuitPermeability final : public Permeability
+{
+public:
+    explicit DupuitPermeability(
+        ParameterLib::Parameter<double> const& permeability_parameter,
+        int const dimension)
+        : Permeability(permeability_parameter, dimension)
+    {
+    }
+
+    ~DupuitPermeability() = default;
+
+    /**
+     *  Get property value.
```

Comment:
It's not property value, it's the permeability tensor. If possible, do not duplicate docu from the base class; doxygen should be doing this.

Reply:
Referenced the (updated) base class documentation.